### PR TITLE
chore(deps): Added readme tag to update readme page

### PR DESCRIPTION
**Pull request by SimonDarksideJ**
Updated the readme tag for the UIExtensions package

### CODE REVIEW

File: data/packages/com.unity.uiextensions.yml [hunk 1]
```diff
@@ -2,6 +2,7 @@ name: com.unity.uiextensions
 displayName: Unity UI Extensions
 description: The Unity UI Extensions project is a collection of extension scripts / effects and controls to enhance your Unity UI experience. These scripts have been gathered from many sources, combined and improved over time.
 repoUrl: 'https://github.com/Unity-UI-Extensions/com.unity.uiextensions'
+readme: 'upm:Packages/com.unity.uiextensions/README.md'
```
Comment (favoyang) [inline data/packages/com.unity.uiextensions.yml:5]:
```
readme: 'upm:Packages/com.unity.uiextensions/README.md'
```
`upm` - is the branch
`Packages/com.unity.uiextensions/README.md` is the path of the README.md file

In this case, it should be 

```
readme: 'release:README.md'
```

**Review by favoyang — CHANGES_REQUESTED** [review]
Please fix the review comment.

File: data/packages/com.unity.uiextensions.yml (hunk 1 above)
Comment (SimonDarksideJ) [reply data/packages/com.unity.uiextensions.yml:5]: righteo, will get that updated asap, thanks

**Comment by SimonDarksideJ** [comment]
Comments addressed

**Review by favoyang — APPROVED** [review]

**Pull request merged**